data_steward/cdr_cleaner/clean_cdr_engine.py

```diff
@@ -108,7 +135,30 @@ def clean_dataset(project=None, statements=None, data_stage=stage.UNSPECIFIED):
                        data_stage)
 
     if failures > 0:
```

Comment:
nit:  After running the pipeline with Aimee, it would be nice if this conditional had an else statement that explicitly said there were no failures.

Reply:
Added the else statement.